openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaClientGenerator.java

```diff
@@ -189,6 +205,9 @@ public SyntaxTree generateSyntaxTree() throws BallerinaOpenApiException {
         }
         // Add class definition node to module member nodes
         nodes.add(getClassDefinitionNode());
+        if (ballerinaAuthConfigGenerator.isApiKey() && ballerinaAuthConfigGenerator.isHttpOROAuth()) {
```

Comment:
Shall we add `AuthConfigRecord` before adding the ClassDefinitionNode since other records are generated before the class definition ?

Reply:
Done